website/docs/cloud-docs/agents/index.mdx

```diff
@@ -181,7 +181,7 @@ export TFC_AGENT_NAME=your-agent-name
 ./tfc-agent
 ```
 
--> **Note:** The `TFC_AGENT_NAME` variable is optional. If you do not specify a name here, one will not be displayed. These names are for your reference only, and the agent ID is what will appear in logs and API requests.
+-> **Note:** The `TFC_AGENT_NAME` variable is optional. If you do not specify a name here, one will not be displayed. These names are for your reference only, and the agent ID is what will appear in logs and API requests.The limit for the name value is 64 characters. More will result in an error.
```

Comment:
Looks like this may need a space after the period at the change start. Also, the last sentence is a fragment. Perhaps, it should be "A character count above that will result in an error."